Mobile Buy SDK/Mobile Buy SDK/Buy.h

```diff
@@ -65,6 +65,7 @@ FOUNDATION_EXPORT const unsigned char BuyVersionString[];
 #import <Buy/BUYClient.h>
 #import <Buy/BUYClient+Customers.h>
 #import <Buy/BUYClient+Checkout.h>
+#import <Buy/BUYClient+CheckoutHelpers.h>
```

Comment:
What is the need for exposing this category? It has only a single method `- (void)handleCheckout:completion:`, which in my opinion is really vague and confusing. If we expose this as public API than there's really no need for this to be in a separate file / category, and I think it should be renamed to something that describes its purpose better.